File: stepik-java-api/src/main/java/org/stepik/api/actions/StepikViewsAction.java
package org.stepik.api.actions;

import org.jetbrains.annotations.NotNull;
import org.stepik.api.client.StepikApiClient;
import org.stepik.api.queries.views.StepikViewsPostQuery;

/**
 * @author meanmail
 */
public class StepikViewsAction extends StepikAbstractAction {
    public StepikViewsAction(@NotNull StepikApiClient stepikApiClient) {
        super(stepikApiClient);
    }

    @NotNull
    public StepikViewsPostQuery post() {
        return new StepikViewsPostQuery(this);
    }

}

File: stepik-java-api/src/main/kotlin/org/stepik/api/actions/StepikViewsAction.kt
package org.stepik.api.actions

import org.stepik.api.client.StepikApiClient
import org.stepik.api.queries.views.StepikViewsPostQuery

class StepikViewsAction(stepikApiClient: StepikApiClient) : StepikAbstractAction(stepikApiClient) {
    
    fun post(): StepikViewsPostQuery {
        return StepikViewsPostQuery(this)
    }
    
}
